python/dgl/geometry/mxnet/fps.py

```diff
@@ -44,10 +46,13 @@ def forward(self, pos, random_start=True):
         B, N, C = pos.shape
         pos = pos.reshape(-1, C)
         dist = nd.zeros((B * N), dtype=pos.dtype, ctx=ctx)
-        if random_start:
+        if start_idx is None:
             start_idx = nd.random.randint(0, N - 1, (B, ), dtype=np.int, ctx=ctx)
         else:
-            start_idx = nd.zeros((B, ), dtype=np.int, ctx=ctx)
+            if start_idx > N:
+                raise DGLError("Invalid start_idx, expected index <= {}, got {}".format(
```

Comment:
and the error message needs to change accordingly.

Reply:
solved

Reply:
I mean the message needs to be something like `expected 0 <= index < {}`.

Reply:
oh, my fault